kubectl-minio/cmd/tenant-create.go

```diff
@@ -61,15 +64,14 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 		Long:    createDesc,
 		Example: createExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := c.validate(); err != nil {
+			if err := c.validate(args); err != nil {
 				return err
 			}
 			return c.run(args)
 		},
 	}
-
+	cmd = helpers.DisableHelp(cmd)
 	f := cmd.Flags()
-	f.StringVar(&c.tenantOpts.Name, "name", "", "name of the MinIO tenant to create")
 	f.Int32Var(&c.tenantOpts.Servers, "servers", 0, "total number of pods in MinIO tenant")
```

Comment:
If this is help text (i think it is), can we add something like "(Required)" trailing each, so users can see from help these are all required flags?